metabase/metabase
# Add driver warning to the new database form

Epic https://github.com/metabase/metabase/issues/26178

How to test:
- `yarn storybook` -> `DatabaseForm`
- Select `Presto` driver
- Make sure you can change to the deprecated driver and vice versa via the warning

<img width="1499" alt="Screenshot 2022-11-29 at 16 40 55" src="https://user-images.githubusercontent.com/8542534/204558891-298c6477-4d2a-4fd7-8734-60536493ac9b.png">
